diff --git a/node_modules/@remix-run/server-runtime/routeModules.d.ts b/node_modules/@remix-run/server-runtime/routeModules.d.ts
index ae9c71f..70a3ede 100644
--- a/node_modules/@remix-run/server-runtime/routeModules.d.ts
+++ b/node_modules/@remix-run/server-runtime/routeModules.d.ts
@@ -18,8 +18,8 @@ export interface DataFunctionArgs {
 /**
  * A function that handles data mutations for a route.
  */
-export interface ActionFunction {
-    (args: DataFunctionArgs): Promise<Response> | Response | Promise<AppData> | AppData;
+export interface ActionFunction<AppData = null> {
+    (args: DataFunctionArgs): Promise<AppData> | AppData;
 }
 /**
  * A React component that is rendered when the server throws a Response.
@@ -52,17 +52,17 @@ export interface LinksFunction {
 /**
  * A function that loads data for a route.
  */
-export interface LoaderFunction {
-    (args: DataFunctionArgs): Promise<Response> | Response | Promise<AppData> | AppData;
+export interface LoaderFunction<AppData = null> {
+    (args: DataFunctionArgs): Promise<AppData> | AppData;
 }
 /**
  * A function that returns an object of name + content pairs to use for
  * `<meta>` tags for a route. These tags will be merged with (and take
  * precedence over) tags from parent routes.
  */
-export interface MetaFunction {
+export interface MetaFunction<LoaderData = null> {
     (args: {
-        data: AppData;
+        data?: LoaderData;
         parentsData: RouteData;
         params: Params;
         location: Location;
