diff --git a/node_modules/@remix-run/server-runtime/dist/routeModules.d.ts b/node_modules/@remix-run/server-runtime/dist/routeModules.d.ts
index 229aa96..d0a1f8e 100644
--- a/node_modules/@remix-run/server-runtime/dist/routeModules.d.ts
+++ b/node_modules/@remix-run/server-runtime/dist/routeModules.d.ts
@@ -1,3 +1,4 @@
+import { UseDataFunctionReturn } from "@remix-run/react/dist/components";
 import type { Location } from "history";
 import type { ComponentType } from "react";
 import type { Params } from "react-router-dom";
@@ -62,9 +63,9 @@ export interface LoaderFunction {
  * `<meta>` tags for a route. These tags will be merged with (and take
  * precedence over) tags from parent routes.
  */
-export interface MetaFunction {
+export interface MetaFunction<T = AppData> {
     (args: {
-        data: AppData;
+        data?: UseDataFunctionReturn<T>;
         parentsData: RouteData;
         params: Params;
         location: Location;
