diff --git a/node_modules/@remix-run/react/dist/esm/transition.js b/node_modules/@remix-run/react/dist/esm/transition.js
index 9923a7c..32d2396 100644
--- a/node_modules/@remix-run/react/dist/esm/transition.js
+++ b/node_modules/@remix-run/react/dist/esm/transition.js
@@ -1162,11 +1162,10 @@ function findNearestBoundary(matchWithError, matches) {
 function makeLoaderData(state, results, matches) {
   let newData = {};
 
-  for (let {
-    match,
-    value
-  } of results) {
-    newData[match.route.id] = value;
+  for (let result of results) {
+    if (!isCatchResult(result) && !isErrorResult(result)) {
+      newData[result.match.route.id] = result.value;
+    }
   }
 
   let loaderData = {};
