diff --git a/node_modules/@remix-run/node/parseMultipartFormData.js b/node_modules/@remix-run/node/parseMultipartFormData.js
index 0459664..cab4a15 100644
--- a/node_modules/@remix-run/node/parseMultipartFormData.js
+++ b/node_modules/@remix-run/node/parseMultipartFormData.js
@@ -83,7 +83,7 @@ async function internalParseFormData(contentType, body, abortController, uploadH
             // *after* busboy has finished. Rethrow the error for surfacing
             // in the Promise.all(fileWorkQueue) below.
 
-            throw error;
+            // throw error;
           } finally {
             filestream.resume();
           }
