diff --git a/node_modules/randomfill/browser.js b/node_modules/randomfill/browser.js
index ce34a69..c880253 100644
--- a/node_modules/randomfill/browser.js
+++ b/node_modules/randomfill/browser.js
@@ -36,7 +36,7 @@ function assertSize (size, offset, length) {
     throw new RangeError('buffer too small')
   }
 }
-if ((crypto && crypto.getRandomValues) || !process.browser) {
+if ((crypto && crypto.getRandomValues) || !true) {
   exports.randomFill = randomFill
   exports.randomFillSync = randomFillSync
 } else {
@@ -64,7 +64,7 @@ function randomFill (buf, offset, size, cb) {
 }
 
 function actualFill (buf, offset, size, cb) {
-  if (process.browser) {
+  if (true) {
     var ourBuf = buf.buffer
     var uint = new Uint8Array(ourBuf, offset, size)
     crypto.getRandomValues(uint)
